src/pages/my_registrations/RegistrationList.tsx

```diff
@@ -85,7 +85,7 @@ const RegistrationList: FC<RegistrationListProps> = ({ registrations }) => {
             {registrations.map((registration) => (
               <StyledTableRow key={registration.identifier}>
                 <TableCell component="th" scope="row">
-                  <Typography>{registration.mainTitle}</Typography>
+                  <Typography>{registration.mainTitle ?? <i>[{t('common:missing_title')}]</i>}</Typography>
```

Comment:
Egentlig bør vel ikke italic-taggen brukes til ren stiling. men det er jo kort og oversiklig da :-)

Reply:
har egnetlig ikke helt skjønt hvordan MUI vil at man skal gjøre det det... å legge det til styled-compoenents e.l. er bare unødvendig styr spør du meg, så har de eksempel i docs hvor de bruker en egen komponent for bare dette: https://material-ui.com/system/typography/#font-style
skjønner ikke hvorfor `Typography` ikke bare kan ta inn en prop ``fontStyles` i stedet